js/ascendex.js

```diff
@@ -1703,6 +1704,112 @@ module.exports = class ascendex extends Exchange {
         return await this.v2PrivateAccountGroupGetFuturesPosition (this.extend (request, params));
     }
 
+    parseFundingRate (fundingRate, market = undefined) {
+        //
+        // {
+        //     time: "1640078764560",
+        //     symbol: "BTC-PERP",
+        //     markPrice: "48499.644083351",
+        //     indexPrice: "48530.35",
+        //     openInterest: "252.1125",
+        //     fundingRate: "-0.000631045",
+        //     nextFundingTime: "1640102400000"
+        // }
+        //
+        const marketId = this.safeString (fundingRate, 'symbol');
+        const symbol = this.safeSymbol (marketId, market);
+        const currentTime = this.safeInteger (fundingRate, 'time');
+        const nextFundingRate = this.safeNumber (fundingRate, 'fundingRate');
+        const nextFundingRateTimestamp = this.safeInteger (fundingRate, 'nextFundingTime');
+        let previousFundingTimestamp = undefined;
+        if (nextFundingRateTimestamp !== undefined) {
+            previousFundingTimestamp = nextFundingRateTimestamp - 28800000;
+        }
+        return {
+            'info': fundingRate,
+            'symbol': symbol,
+            'markPrice': this.safeNumber (fundingRate, 'markPrice'),
+            'indexPrice': this.safeNumber (fundingRate, 'indexPrice'),
+            'interestRate': this.parseNumber ('0'),
+            'estimatedSettlePrice': undefined,
+            'timestamp': currentTime,
+            'datetime': this.iso8601 (currentTime),
+            'previousFundingRate': undefined,
+            'nextFundingRate': nextFundingRate,
+            'previousFundingTimestamp': previousFundingTimestamp,
+            'nextFundingTimestamp': nextFundingRateTimestamp,
+            'previousFundingDatetime': this.iso8601 (previousFundingTimestamp),
+            'nextFundingDatetime': this.iso8601 (nextFundingRateTimestamp),
+        };
+    }
+
+    async fetchFundingRate (symbol, params = {}) {
+        if (symbol === undefined) {
```

Comment:
In this case, the `symbol` argument does not have a default value, so we don't have to do the check for `symbol === undefined` here, and we should not throw an exception.
Basically, the rule of thumb is: 
1. if an argument has a default value of `undefined` according to the unified method signature (the majority of methods with the same name across the lib), then we add a check for `=== undefined` and then we have to throw an exception.
2. otherwise, if an argument (`symbol`) in this case does not have a default value, then the user is required to supply it, and we don't have to check for and we don't throw a exception on it